src/main/java/io/jenkins/plugins/orka/helpers/FormValidator.java

```diff
@@ -99,7 +99,7 @@ public FormValidation doCheckMemory(String memory) {
             if (StringUtils.isBlank(memory) || StringUtils.equals(memory, "auto")) {
                 return FormValidation.ok();
             }
-            if (Integer.parseInt(memory) < 1) {
+            if (Float.parseFloat(memory) < 0) {
                 return FormValidation.error("Memory should be greater than 0");
```

Comment:
Error will not be thrown if `memory = 0`

Reply:
Just wondering if this accounts for 0 setting the RAM to auto calculation. 

We should try to match Orka feature set if set to 0 or left blank it should result in auto calibration of the RAM.

Reply:
Auto can be set in two ways - leaving the field blank or typing auto.

As for @bwilliams7 comment - yes, we should not allow the memory to be 0.

Reply:
In that case, I'll be a bit pedantic here and say the text of the error should be changed from
`Memory should be greater than 0`
to
`Memory should be 0 or greater`

Reply:
I see this as resolved, but do not see any changes :)